Website/src/rendering/src/components/HeroSection.tsx

```diff
@@ -1,50 +1,40 @@
-import { ComponentProps } from 'lib/component-props';
-import { Text, Field, LinkField, Link, ImageField, Image } from '@sitecore-jss/sitecore-jss-nextjs';
+import { Text, Field, RichText, ImageField } from '@sitecore-jss/sitecore-jss-nextjs';
+import { ComponentWithChildrenProps } from 'lib/component-props';
+import { Placeholder } from '@sitecore-jss/sitecore-jss-nextjs';
 
-export type HeroProps = ComponentProps & {
+export type HeroProps = ComponentWithChildrenProps & {
```

Comment:
What is the reason for this change of base props? I do not see `props.children` used anywhere in this component.